scripts/linters/linter_utils.py

```diff
@@ -122,3 +123,24 @@ def redirect_stdout(new_target):
         yield new_target
     finally:
         sys.stdout = old_target
+
+
+def get_duplicates_from_list_of_strings(strings):
+    """Returns a list of duplicate strings in the list of given strings.
+
+    Args:
+        strings: list(str). A list of strings.
+
+    Returns:
+        list(str). A list of duplicate string present in the given list of
+        strings.
+    """
+    duplicates = []
+    item_count_map = collections.defaultdict(int)
+    for string in strings:
+        item_count_map[string] += 1
+        # Counting as duplicate once its appeared twice in the list.
```

Comment:
its --> it's

(Note: it's == "it is" or "it has")

Reply:
(Ah, my bad!) Fixed